src/simulator_adapter.py

```diff
@@ -228,6 +238,8 @@ def obstacle_odom_callback(msg, topic, data={}):
     accel_pub = rospy.Publisher('acceleration', Vector3Stamped, queue_size=0)
     battery_pub = rospy.Publisher('fc_battery', Float32, queue_size=0)
     status_pub = rospy.Publisher('fc_status', FlightControllerStatus, queue_size=0)
+    if publish_ground_truth_localization:
```

Comment:
I organized these by who they're communicating with (publishers to morse, subscribers from morse, publishers to the rest of the world, subscribers from the rest of the world), so don't want to change that in this PR, but this node could definitely be reorganized in the future